MinecraftClient/Settings.cs

```diff
@@ -601,6 +603,7 @@ public static void WriteDefaultSettings(string settingsfile)
                 + "scriptcache=true                   # Cache compiled scripts for faster load on low-end devices\r\n"
                 + "timestamps=false                   # Prepend timestamps to chat messages\r\n"
                 + "autorespawn=false                  # Toggle auto respawn if client player was dead (make sure your spawn point is safe)\r\n"
+                + "consoleiocolors=true               # Strips the color codes from output when in BasicIO mode\r\n"
```

Comment:
Using a setting may pose an issue with programs and scripts calling MCC and expecting color codes to be present. I'd suggest to use a command-line argument (e.g. `BasicIO-NoColors`) instead and set the setting directly inside the ConsoleIO class 🙂